config.go

```diff
@@ -43,6 +43,10 @@ func parseFlags() {
 	flag.String("clientaddress", defaultClientAddress, "Address to bind the REST service.")
 	flag.String("peeraddress", defaultPeerAddress, "Address to bind the inter glusterd2 RPC service.")
 
+	// TODO: SSL/TLS only implemented for REST interface
+	flag.String("client-cert-file", "", "Certificate used for SSL/TLS connections from clients.")
```

Comment:
client-cert implies the cert is of the clients connecting to this server. IMO, a better name would be `tls-cert` and `tls-key`.